src/store/overlay/overlay.epics.ts

```diff
@@ -359,9 +370,18 @@ const notifyHostGPTMessageStatus: AppEpic = (_, state$) =>
   );
 
 // models are loading after conversations, if models loaded that means that we can work with application. Maybe there is better condition.
-const notifyHostAboutReadyEpic: AppEpic = (action$) =>
-  action$.pipe(
-    filter(ModelsActions.getModelsSuccess.match),
+const notifyHostAboutReadyEpic: AppEpic = (_action$, state$) =>
+  state$.pipe(
+    filter((state) => {
+      const isAuthDisabled = SettingsSelectors.selectIsAuthDisabled(state);
+      const isShouldLogin = AuthSelectors.selectIsShouldLogin(state);
+      const authStatus = AuthSelectors.selectStatus(state);
+      if (isAuthDisabled || (authStatus !== 'loading' && isShouldLogin)) {
```

Comment:
Does it make sense to create constants for `loading`?

Reply:
Not sure, status field is more flexible in such cases